### ISSUE TITLE
admin and editor should be separate roles with separate permissions

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#297]
"Editor" and "Admin" should separate roles with separate permissions. Broadly the admin role should grant the user permissions to manage users , while the editor role should grant the user permissions to manage the project workflow.

**Comment by elfeto** [comment]
I think the admin should be able to do anything, and the editor only the project workflow.

**Comment by tompollard** [comment]
Yep, you are right...makes sense for admin to be all. Perhaps need another role then for managing users.
